package.json

```diff
@@ -46,18 +46,21 @@
     "electron-log": "^2.2.12",
     "electron-updater": "^2.23.3",
     "electron-window-state": "^4.1.1",
+    "express": "^4.16.4",
     "filewatcher-webpack-plugin": "^1.2.0",
     "find-process": "^1.1.0",
     "formik": "^0.10.4",
     "hast-util-sanitize": "^1.1.2",
-    "keytar": "^4.2.1",
+    "keytar": "^4.3.0",
+    "lbry-format": "https://github.com/lbryio/lbry-format.git",
     "lbry-redux": "lbryio/lbry-redux#820b6eeadf9e58c1e5027fd4d92808ba817b410e",
     "lbryinc": "lbryio/lbryinc#68c8ce6b7608dabc9180fff9b4841d10e1c62284",
     "localforage": "^1.7.1",
     "mammoth": "^1.4.6",
     "mime": "^2.3.1",
     "mixpanel-browser": "^2.17.1",
     "moment": "^2.22.0",
+    "node-fetch": "^2.3.0",
```

Comment:
We aleady bring in `axios` for fetching in the build process. I'm in favor of using `node-fetch` for the main process, it might help to dump axios while were adding this.
https://github.com/lbryio/lbry-desktop/blob/master/build/downloadDaemon.js#L46

Reply:
I am in favor of this, do we want to do this now? (upcoming release, at the edge of release)
That entire function could use a rewrite to `async`/`await`.

Reply:
I think we might as well do it now. the `async/await` stuff might add a lot of work though. It should be super simple to swap axios for fetch in `downloadDaemon.js`